activity/src/main/java/com/amazonaws/stepfunctions/cloudformation/activity/ResourceModelUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.amazonaws.stepfunctions.cloudformation.activity;
+
+import com.amazonaws.services.stepfunctions.model.DescribeActivityResult;
+import com.amazonaws.services.stepfunctions.model.Tag;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ResourceModelUtils {
+
+    /**
+     * Generates a resource model containing all activity resource properties
+     * @param describeActivityResult The result of calling DescribeActivity for the activity
+     * @param activityTags A list of tags associated with the activity
+     * @return A resource model containing the ARN, Name, and list of tags for the activity
+     */
+    public static ResourceModel getUpdatedResourceModelFromReadResults(DescribeActivityResult describeActivityResult,
+                                                          List<Tag> activityTags) {
+        ResourceModel model = new ResourceModel();
+
+        model.setArn(describeActivityResult.getActivityArn());
+        model.setName(describeActivityResult.getName());
+        model.setTags(getTagsEntriesFromTags(activityTags));
```

Comment:
I thought the point of this change is to make the resource model have every field of the Describe + ListTagsForResource result? `creationDate` is missing here.

Reply:
The point of the change is to generate a resource model that:

- conforms to the resource schema
- contains all properties from the resource schema that are defined in the Read API results.